web3/hardhat/contracts/CrowdfundFactory.sol

```diff
@@ -0,0 +1,16 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0 <0.9.0;
+
+import { Crowdfund } from "./Crowdfund.sol";
+
+contract CrowdfundFactory {
+    uint256 public totalCrowdfunds;
+
+    event crowdfundCreated(address creator, Crowdfund crowdfund);
+
+    function createCrowdfund(bytes calldata meta) public {
+        Crowdfund _crowdfund = new Crowdfund(meta);
```

Comment:
Are proxy clones less gas to deploy?

Reply:
Proxy clones are, yes. Initial PoC not rushing to optimize rn